thebeetoken/beenest-web
# Retain state in Close To... filter

## Description
Follows up on #274; retains the user's place selection in the "Close To..." dropdown.

## How to Test
1. Go to `/work` and search
2. Enter a place in the the "Close To..." dropdown
3. Verify listings are still requeried with sort order relative to that place
4. Verify that place remains selected in the dropdown

Which devices did you test on?

## REVIEWERS:
Check against these principles:

### High level
Does this code need to be written?
What are the alternatives?
Will this implementation become a support issue?
How much error margin does this solution have?

### Code
* Does the code follow industry standards?
JS: https://github.com/airbnb/javascript
React: https://github.com/airbnb/javascript/tree/master/react
https://github.com/vasanthk/react-bits
Documentation headers: http://usejsdoc.org/index.html

* Is there duplicated code? Can it be refactored into a shared method?
* Is the code consistent with our project?
* Are there unit tests? Do they test the states?
* Is the person refactoring another developer's code? If possible, did the original developer approve?

### Variables/Naming:
* Would the variable type led to future edge cases?
* Are the variable naming clear? Would the value contain something other than what the name describes.

### Security
* Can this be hacked or abused by the user?

## Further Work
1. Show distinct pin at destination
2. Display driving directions
3. Moar filters
